### ISSUE TITLE
Tab sidebar slideout low fps

### ISSUE THREAD

**Issue by agn4965** [anthonycr/Lightning-Browser#188]
The difference between a tab sidebar at 8 tabs and the bookmarks sidebar at 12 bookmarks is too significant. The tab sidebar stutters as it comes out when there's more than a few tabs active, the fading to black of the rest of the screen is also affected by it, not smooth at all.

'm using lightning lollipop beta 2 with my moto g2 on 5.0.2

**Comment by anthonycr** [comment]
I too noticed this, what's weird is that it wasn't present before the lollipop updates I made and I honestly didn't change too much although I did update the underlying drawer widget. I just set the layer type to use hardware acceleration and voila the lag goes away... it's like the newer support v4 drawer I was using disabled hardware acceleration.

What's also odd is that no such problem exists with the bookmark drawer and it is similar in layout. Anyway, we'll see if the fix I introduce works for the next update.
